style/js/shield_defs.js

```diff
@@ -1,619 +1,634 @@
-    backgroundImage: [
-      shieldImages.shield40_it_2,
-      shieldImages.shield40_it_3,
-      shieldImages.shield40_it_4,
-    ],
-    textColor: "white",
-    padding: {
-      left: 2,
-      right: 2,
-      top: 5,
-      bottom: 5,
-    },
-  };
-
-  // Switzerland
-  shields["motorway"] = {
-    backgroundImage: shieldImages.shield40_ch_2,
-    textColor: "white",
-    padding: {
-      left: 2,
-      right: 2,
-      top: 5,
-      bottom: 5,
-    },
-  };
-
-  shields["HU:national"] = {
-    backgroundImage: shieldImages.shield40_hu_2,
-    textColor: "white",
-    padding: {
-      left: 3,
-      right: 3,
-      top: 2,
-      bottom: 6,
-    },
-  };
-
-  shields["sk:national"] = roundedRectShield(
-    "#ba1e10",
-    "white",
-    "white",
-    3,
-    2,
-    33
-  );
-
-  shields["SI:AC"] = {
-    backgroundImage: shieldImages.shield40_si_2,
-    textColor: "white",
-    padding: {
-      left: 3,
-      right: 3,
-      top: 4,
-      bottom: 5,
-    },
-  };
-
-  // Croatia
-  shields["Autoceste"] = shields["SI:AC"];
-  shields["Državne ceste"] = shields["cz:national"];
-
-  // Netherlands
-  shields["NL:A"] = roundedRectShield(
-    "#ba1e10",
-    "white",
-    "white",
-    2,
-    1.5,
-    null
-  );
-  shields["NL:N"] = roundedRectShield("#eacb44", null, "black", 2, 0, null);
-
-  return shields;
-}
-
-/**
- * Determines whether there is a raster shield background for a particular network
- *
- * @param {*} network - Route network
- * @returns true if a raster shield is available
- */
-export function hasShieldArtwork(network) {
-  return (
-    shields[network] != null &&
-    typeof shields[network].backgroundImage !== "undefined"
-  );
-}
-
-/**
- * Determines whether a shield should be changed in color from the base color
- * to another color via a "lighten" operation.  For example, Historic US 66
- * should be converted from the standard black US route shield to the historic
- * brown color.  This function is a comprehensive list of special cases.
- *
- * @param {*} network - Route network
- * @param {*} ref - Route ref value
- */
-export function shieldLighten(network, ref) {
-  var shieldDef = shields[network];
-  if (shieldDef == null) {
-    return null;
-  }
-  //Ref-specific cases:
-  switch (network) {
-    case "US:GA":
-      switch (ref) {
-        case "515":
-          return "#003478";
-        case "520":
-          return "#006a4d";
-        default:
-          return null;
-      }
-    default:
-      //Network-specific cases are defined in shield definitions:
-      return shieldDef.colorLighten;
-  }
-}
-
-/**
- * Get the number of banner placards associated with this shield
- *
- * @param {*} shield - Shield definition
- * @returns the number of banner placards that need to be drawn
- */
-export function getBannerCount(shield) {
-  if (shield == null || typeof shield.modifiers == "undefined") {
-    return 0; //Unadorned shield
-  }
-  return shield.modifiers.length;
-}
+"use strict";

```

Comment:
Looks like this PR changed the file to use Windows-style line endings instead of Unix-style line endings. Run `npm run code_format` to change it back and apply any other formatting fixes.

Reply:
Thanks for the advice!  This is addressed in bd0753125d96e3d5af1cb12e0c7e249e7dd3bd38